cmds/ip/ops.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
```

Comment:
2018

Reply:
done

Reply:
It still says 2017... I'll automatically change it later with the script.